style.css

```diff
@@ -1 +1,75 @@
 /*  Add your styling here */
+
+.flex-container {
+  display: flex;
+  background-color: #d3d3d3;
+  justify-content: space-around;
+}
+
+.flex-container > div {
+  background-color: #a9a9a9;
+  margin: 0.627em;
+  padding: 1.255em;
+  font-size: 1.882em;
+}
+
+.nav-bar {
+  display: flex;
+  background-color: #d3d3d3;
+  justify-content: space-around;
+  flex-direction: row-reverse;
+}
+
+.nav-bar > div {
+  background-color: #0a0101;
+  margin: 0.627em;
+  padding: 0.1em;
+  font-size: 1.882em;
+  color: white;
```

Comment:
You've mixed hex colors and web safe colours here. It's best to stick to one format.